src/common-ssh/common-ssh/user.h

```diff
@@ -44,6 +44,12 @@ typedef struct guac_common_ssh_user {
      */
     guac_common_ssh_key* private_key;
 
+    /**
+     * The public key which should be used to authenticate this user, if any,
+     * or NULL if a password or just a private key will be used instead.
+     */
+    char* public_key;
```

Comment:
Is "public key" the correct term for this? To me, a "public key" is the half of a key-pair that you share with someone in order to connect or facilitate encryption. Per the Jira issue, shouldn't this be `ssh_certificate` or something like that?

Reply:
@necouchman It is a public key but a CA signed public key. But, I kept in generic. Should I rename to ca_public_key instead?

Reply:
Okay, I see - I had to read the the Red Hat page linked on the Jira issue to actually understand the point of what was going on, here. I think the naming as you have it is fine.